lib/bamboozled/api/time_off.rb

```diff
@@ -14,7 +13,9 @@ def requests(options = {})
         # Make sure all statuses are allowed
         if options[:status]
           allowed_statuses = %w(approved denied superceded requested canceled)
-          options[:status] = Array(options[:status]).keep_if { |v| allowed_statuses.include? v }.join(",")
+          options[:status] = Array(options[:status])
+                             .keep_if { |v| allowed_statuses.include? v }
```

Comment:
Hound is using the old rubocop config still. Once merged this will be valid style.